core/src/main/java/com/distkv/dst/core/KVStore.java

```diff
@@ -0,0 +1,27 @@
+package com.distkv.dst.core;
+
+import com.distkv.dst.core.operatorset.DstDict;
+import com.distkv.dst.core.operatorset.DstList;
+import com.distkv.dst.core.operatorset.DstSet;
+import com.distkv.dst.core.operatorset.DstString;
+import com.distkv.dst.core.operatorset.DstSortedList;
+import com.distkv.dst.core.operatorset.DstTable;
+
+/**
+ * The api of dst store.
+ */
+public interface KVStore {
+
+  DstString strs();
+
+  DstList lists();
+
+  DstSet sets();
+
+  DstDict dicts();
+
+  DstSortedList sortLists();
+
+  DstTable tables();
+
+}
```

Comment:
This file shouldn't be a new file. U have to modify in the original file.

Reply:
I don't seem to see any difference from the original file. What is difference ?